api-sdk/src/main/java/com/smartling/api/sdk/file/parameters/FileUploadParameterBuilder.java

```diff
@@ -110,6 +111,18 @@ public FileUploadParameterBuilder fileType(final FileType fileType)
         return this;
     }
 
+    /**
+     * library UID that uses sdk for sending files
+     * if UID is not provided default UID that is equal to sdk library UID is set
+     * @param clientUid
+     * @return
+     */
+    public FileUploadParameterBuilder clientUid(final String clientUid)
```

Comment:
If this ID has a strict format, should not we change method signature to smth like `clientInfo(String name, String version)` and build JSON internally? Because with a current design any connector will have to do the same.